docs/kratos/social-signin/09_ory.mdx

```diff
@@ -0,0 +1,182 @@
+      // verification or as a password login identifier.
+      //
+      // Therefore we only return the email if it (a) exists and (b) is marked verified.
+      [if 'email' in claims && claims.email_verified then 'email' else null]: claims.email,
+    },
+  },
+}
+```
+
+```mdx-code-block
+<JsonnetWarning format="Jsonnet code snippets" use="data mapping" />
+```
+
+</TabItem>
+<TabItem value="cli" label="Ory CLI">
+
+Follow these steps to add a Ory as a social sign-in provider to your project using the Ory CLI:
+
+1. Create a client at your Ory to get a Client ID and Client Secret.
+2. Create a [Jsonnet code snippet](#data-mapping) to map the desired claims to the Ory Identity schema, such as:
+```jsonnet
+local claims = {
+  email_verified: false,
+} + std.extVar('claims');
+
+{
+  identity: {
+    traits: {
+      // Allowing unverified email addresses enables account
+      // enumeration attacks,  if the value is used for
+      // verification or as a password login identifier.
+      //
+      // Therefore we only return the email if it (a) exists and (b) is marked verified.
+      [if 'email' in claims && claims.email_verified then 'email' else null]: claims.email,
+    },
+  },
+}
+```
+3. Encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or host it under an URL accessible to Ory Network.
+4. Download the Ory Identities config from your project and save it to a file:
+
+   ```shell
+   ## List all available projects
+   ory list projects
+
+   ## Get config
+   ory get identity-config {project-id} --format yaml > identity-config.yaml
+   ```
+
+5. Add the social sign-in provider configuration to the downloaded config. Add the Jsonnet snippet with mappings as a Base64
+   string or provide an URL to the file.
+
+   ```yaml
+   selfservice:
+     methods:
+       oidc:
+         config:
+           providers:
+             - id: yourid # this is `<provider-id>` in the Authorization callback URL. DO NOT CHANGE IT ONCE SET!
+               provider: generic
+               client_id: .... # Replace this with the Client ID
+               client_secret: .... # Replace this with the Client secret
+               issuer_url: https://your-project-slug.projects.oryapis.com # Replace this with the providers issuer URL
+               mapper_url: "base64://{YOUR_BASE64_ENCODED_JSONNET_HERE}"
+               # Alternatively, use an URL:
+               # mapper_url: https://storage.googleapis.com/abc-cde-prd/9cac9717f007808bf17f22ce7f4295c739604b183f05ac4afb4
+               scope:
+                 - email
+                 - offline_access
+                 - openid
+               # supported scopes can be found in your providers dev docs
+         enabled: true
+   ```
+
+6. Update the Ory Identities configuration using the file you worked with:
+
+   ```shell
+   ory update identity-config {project-id} --file identity-config.yaml
+   ```
+
+</TabItem>
+</Tabs>
+````
```

Comment:
A closing section with a recap of the end-to-end picture would be nice here

Reply:
you mean a section that explains how the flow works from the user perspective? or a more "architecture" diagram that shows how the projects and frontend are wired up together?